src/transform/inverse.rs

```diff
@@ -810,10 +812,10 @@ trait InvTxfm2D: Dim {
     Self::inv_txfm2d_add_rs(input, output, stride, tx_type, bd);
   }
 
-  fn inv_txfm2d_add_rs(
-    input: &[i32], output: &mut [u16], stride: usize, tx_type: TxType,
+  fn inv_txfm2d_add_rs<T>(
```

Comment:
Do you know where this function figure out the size of tx?  Would Self::W and Self::H be that info?

Reply:
Yes, exactly.